tool/codeviewer_cli/segment_generator.dart

```diff
@@ -19,6 +22,8 @@ const _globalPrologue =
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// @dart=2.9
```

Comment:
@guidezpl , added dart version here in generator until code_style.dart is not migrated, to fix analyzer issue